mlrun/datastore/sources.py

```diff
@@ -209,25 +208,17 @@ def to_step(self, key_field=None, time_field=None, context=None):
         )
 
     def get_spark_options(self):
+        store, path = mlrun.store_manager.get_or_create_store(self.path)
+        result = {
+            "path": store.spark_url + path,
+            "format": "csv",
+            "header": "true",
+            "inferSchema": "true",
+        }
         if self.path and self.path.startswith("ds://"):
```

Comment:
I don't think we need this 'ds' special case anymore. We should always return {**result, **storage_spark_options}. The same for all the further cases